transport-classes-io_uring/src/main/java/io/netty/channel/uring/IoUring.java

```diff
@@ -232,6 +245,22 @@ public static boolean isRegisterBufferRingIncSupported() {
         return IORING_REGISTER_BUFFER_RING_INC_SUPPORTED;
     }
 
+    static boolean isAcceptMultishotEnabled() {
```

Comment:
Perhaps making there methods to public could assist user in better adapting to different runtime environments
such as 
```java
if (!IoUinrg.isSpliceSupported) {
  pipeline.addlast(chunHandler);
}

if(IoUring.isRecvMultishotEnabled) {
  registerBufferRing(...)
}
```

Reply:
This makes sense... Let's make them public.